S/serd/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "serd"
```

Comment:
Last comment: we prefer to match the name of the upstream library, including the case, which seems to be
```suggestion
name = "Serd"
```
Note that you'll need to rename also the directory accordingly

Reply:
Makes sense. I was wondering myself whether I had got the name wrong.